packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -241,7 +251,14 @@ const nuxScreens = (Navigator: typeof Stack) => (
 const sendScreens = (Navigator: typeof Stack) => (
   <>
     <Navigator.Screen name={Screens.Send} component={Send} options={sendScreenNavOptions} />
-    <Navigator.Screen name={Screens.QRNavigator} component={QRNavigator} options={noHeader} />
+    <Navigator.Screen
+      name={Screens.QRNavigator}
+      component={QRNavigator}
+      options={{
+        ...noHeader,
+        ...TransitionPresets.ModalSlideFromBottomIOS,
```

Comment:
Should be `ModalTransition` which under the hood uses `ModalSlideFromBottomIOS` on iOS and `RevealFromBottomAndroid` / `FadeFromBottomAndroid` on Android.